src/App/screens/Instructor/state/epics/fetchTechnologies.js

```diff
@@ -1,16 +1,22 @@
+import {includes} from 'lodash'
 import {Observable} from 'rxjs'
-import {addNotification} from '../../../../state/actions'
+import headers from '../../../../utils/headers'
+import {loginExpiredDescriptionText} from '../../../../utils/text'
+import {removeUser, addNotification} from '../../../../state/actions'
 import {STARTED_FETCH_TECHNOLOGIES} from '../actions/instructorActionTypes'
 import {endFetchTechnologies} from '../actions'
-import headers from '../../../../utils/headers'
 
 export default (action$, store) => (
   action$.ofType(STARTED_FETCH_TECHNOLOGIES)
     .switchMap(
       ({payload}) => Observable.fromPromise(
         fetch(`${process.env.REACT_APP_EGGHEAD_BASE_URL}/api/v1/technologies`, {headers})
           .then(response => {
-            if (!response.ok) {
+            if (includes([401, 404], response.status)) {
+              store.dispatch(removeUser())
+              throw Error(loginExpiredDescriptionText)
+            }
```

Comment:
I would still like to clean up my epics / fetch logic a bit to reuse but waiting a bit longer to pick the right abstraction